signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupUIBaseBean.java

```diff
@@ -738,11 +754,13 @@ public String getAllAttendeesEmailAddressesFormatted() {
 		Set<String> emails = new HashSet<String>();
 		
 		StringBuilder sb = new StringBuilder();
-		for (TimeslotWrapper tsWrapper : timeslotWrappers) {
-			for(AttendeeWrapper atWrapper : tsWrapper.getAttendeeWrappers()) {
-				String email = atWrapper.getAttendeeEmail();
-				if(StringUtils.isNotBlank(email)){
-					emails.add(email);
+		if (timeslotWrappers!=null){

```

Comment:
The problem is that the details page shows a stacktrace and we don't want it to.  This fixes it by avoiding the NPE that causes the stacktrace.  In any fix someone can change something and break it again.